CHANGELOG.md

```diff
@@ -38,6 +38,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Add documentation for Getting Started section including FAQs
 
 - **CUMULUS-2185** - RDS Migration Epic
+  - **CUMULUS-2188**
+    - Added `data-migration2` Lambda to be run after `data-migration1`
```

Comment:
This isn't specifically a changelog comment, but I think this needs some sort of README in `lambdas/data-migration2` and/or `example/data-migration/*` for devs prior to the user docs.

Reply:
(and yes I realize it might be just "run the lambda")

Reply:
There is a separate ticket, 2293, for writing the docs on how to run the data migrations

Reply:
I don't believe that addresses the concern - I'm not talking about a comprehensive user doc, more of a "this is how we're expecting to run it" 5-6 line cheat sheet.

Reply:
That totally makes sense. I added 2 READMEs, one for each of the data-migration Lambdas. They're pretty light, just answering the question "What the heck is this?" and giving some details on how it can be run.

Reply:
:100:  Thanks  : 893d5be47f939d027d1567bda00dcde8bc08ff5e